packages/@okta/vuepress-site/docs/reference/saml-hook/index.md

```diff
@@ -368,6 +372,53 @@ When modifying an existing assertions statement, the path could begin with `/sub
 }
 ```
 
+## URL format response example
+
+This example displays a `replace` and an `add` op with the URI formatted claim encoded.
```

Comment:
Seems like there should be a plural in here since there are two replace ops?

Reply:
display `replace` and an `add` operations

Reply:
That is true - there are two replace op and one add op in the example! GAH. I will update.